packages/@n8n/nodes-langchain/nodes/llms/N8nLlmTracing.ts

```diff
@@ -38,11 +38,13 @@ export class N8nLlmTracing extends BaseCallbackHandler {
 
 	completionTokensEstimate = 0;
 
-	lastInput: LastInput = {
-		index: 0,
-		messages: [],
-		options: {},
-	};
+	/**
+	 * A queue to map LLM run IDs to input indices.
+	 * Key: Unique identifier for each LLM run (run ID)
+	 * Value: Index of the input corresponding to this run
+	 *
+	 */
+	runsQueue: Record<string, RunDetail> = {};
```

Comment:
Just a quick question regarding the name: since this is a Record type, which doesn't imply an order, do you think Queue is the accurate term? Could a term like runsMap or something similar better reflect the purpose here?